config/initializers/session_store.rb

```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_rws-ruby-sdk-rails-sample_session'
+Rails.application.config.session_store :cookie_store, key: "_rws-ruby-sdk-rails-sample_session"
```

Comment:
__[rubocop]__
* Line is too long. [95/80]

<!-- Comment by [SideCI](https://sideci.com) -->